### DISCUSSION TITLE
What do you think of having no passwords on the site?

### DISCUSSION THREAD

**Discussion by Nautman** [digitalungdom-se/digitalungdom.se#95] [General]
I felt that I should start a discussion and thought this was an interesting topic!

**Comment by Zigolox** [comment]
I feel it can alienate some users which are not used to login in with their mail. But security is more important!

**Comment by charlesmaddock** [comment]
I agree with @Zigolox! It is quite a smooth login process also

**Comment by Zigolox** [comment]
Great!

**Comment by sakerhetspolisen** [comment]
Sorry for being late but I agree too, only using e-mail seems like a modern approach to it!

**Reply by Nautman** [reply]
Haha there's absolutely no need to apologise. The main reason that I posted this was to test out the feature! The subject was chosen arbitrarily :)

Many times when I tag you or post things on GitHub it's no worries if you don't respond.

P.S. To clarify why I'm responding so "quickly" I happened to see the e-mail notification :P
